reviewing/dependencies-hierarchy/src/getTree.ts

```diff
@@ -112,8 +113,7 @@ function getTreeHelper (
       if (circular) {
         dependencies = []
       } else {
-        const requestedDepth = opts.maxDepth
-        dependencies = dependenciesCache.get({ packageAbsolutePath, requestedDepth })
+        dependencies = dependenciesCache.get({ packageAbsolutePath, requestedDepth: childTreeMaxDepth })
```

Comment:
The same `maxDepth` passed to `getTreeHelper` should be used when looking up the cache. I had an off by one error here by passing `opts.maxDepth - 1` to the `getTreeHelper` recursive call, but only using `opts.maxDepth` here.